src/sentry/analytics/events/integration_failed_to_fetch_commit_context_all_frames.py

```diff
@@ -0,0 +1,18 @@
+from sentry import analytics
+
+
+class IntegrationsFailedToFetchCommitContextAllFrames(analytics.Event):
+    type = "integrations.failed_to_fetch_commit_context_all_frames"
+
+    attributes = (
```

Comment:
You can combine these 2 into 1 file like so: https://github.com/getsentry/sentry/blob/master/src/sentry/analytics/events/rule_snooze.py though it's not necessary

Reply:
Thanks for the tip! Do like that better